collectionFunctions.php

```diff
@@ -21,6 +21,8 @@ function displayPoke(Array $pokemon): String {
         $output .= "<li>Sp. Attack: {$poke['spAttack']}</li>";
         $output .= "<li>Sp. Defense: {$poke['spDefense']}</li>";
         $output .= "<li>Speed: {$poke['speed']}</li>";
+        $output .= "<a href='#' class='edit'>Edit</a>";
```

Comment:
theres a lot of php that only needs single quotes here. If it hasn't got a variable use single quotes . :)